torch/_tensor_str.py

```diff
@@ -213,6 +213,7 @@ def get_summarized_data(self):
         else:
             return self
     if self.size(0) > 2 * PRINT_OPTS.edgeitems:
+        self = self.contiguous()
```

Comment:
Should use `self = self.reshape(self.size(0), -1)` instead. See https://github.com/pytorch/pytorch/issues/10402#issuecomment-412174326

Reply:
Or better, just reshape the printed rows.